pkg/crinit/crinit.go

```diff
@@ -28,21 +29,15 @@ import (
 
 // NewClusterregistryCommand creates the `clusterregistry` command.
 func NewClusterregistryCommand(out io.Writer, defaultServerImage, defaultEtcdImage string) *cobra.Command {
-	cmds := &cobra.Command{
+	rootCmd := &cobra.Command{
 		Use:   "crinit",
 		Short: "crinit runs a cluster registry in a Kubernetes cluster",
 		Long:  "crinit bootstraps and runs a cluster registry as a Deployment in an existing Kubernetes cluster.",
-		Run:   runHelp,
 	}
 
-	// From this point and forward we get warnings on flags that contain "_" separators
-	cmds.SetGlobalNormalizationFunc(flag.WarnWordSepNormalizeFunc)
+	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
```

Comment:
I had actually just made a similar change myself to address the lack of V glogs.

Reply:
Yes, I found this by looking at how other k/ command line tools handle this. It's not the prettiest thing, but it seems to be convention at this point.